<?xml version="1.0"?>
<launch>
  <arg name="log" default="S_XX" />
  <node name="baxter"     pkg="i2r_baxter_motion" type="exp.py">
    <param name="emotion" value="False"/>
    <param name="trust_path" value="False" />
  </node>
  <node name="trust"      pkg="i2r_baxter_motion" type="trust_sim.py">
    <param name="log_file" value="$(arg log)_case0.csv"/>
  </node>
  <node name="change_speed"      pkg="i2r_baxter_motion" type="change_speed.py"/>
  <node name="transparency"     pkg="i2r_baxter_motion" type="monitor_trust.py">
    <param name="extra_trust_show" value="False"/>
    <param name="log_file" value="$demo.txt"/>
  </node>

</launch>
